import_cli.adoc

```diff
@@ -0,0 +1,183 @@
+[#importing-a-managed-cluster-with-the-cli]
+= Importing a managed cluster with the CLI
```

Comment:
Since MCE has no UI, I think the "with the CLI" qualification here is unnecessary as we don't have to contrast with another section that would be "with the UI".  Also, strictly speaking, you don't import a managed cluster because the cluster isn't considered managed until the import is done.  So maybe just "Importing a cluster" or "Importing a cluster into the inventory" might work better?  (While "Importing a cluster to bring it under management" might be the most accurate, its seems too wordy).

Reply:
Yeah, i struggle to find a useful phrase here. Maybe "import cluster" since it's not managed yet?